components/properties/src/lib.rs

```diff
@@ -82,6 +82,7 @@
     )
 )]
 
+pub mod bidi;
```

Comment:
Nit: Feature-gate the module with `#[cfg(feature = "bidi")]` (or "unicode-bidi")

Reply:
done